src/test/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyControllerIntTest.kt

```diff
@@ -106,7 +106,7 @@ class LegacyControllerIntTest : SqsIntegrationTestBase() {
       .returnResult(LegacyAdjustment::class.java)
       .responseBody.blockFirst()!!
 
-    assertThat(result).isEqualTo(CREATED_ADJUSTMENT)
+    assertThat(result).isEqualTo(CREATED_ADJUSTMENT.copy(agencyId = null))
     awaitAtMost30Secs untilCallTo { getNumberOfMessagesCurrentlyOnQueue() } matches { it == 0 }
```

Comment:
So there are currently two tests in this class:

create & `create with API agency ID`

So no longer will there be a  `CREATED_ADJUSTMENT` with a null agency ID. So can you default the agency id and remove the `create with API agency ID` test